README.md

```diff
@@ -309,17 +309,78 @@ func main() {
 }
 ```
 
+### Using Paging
+
+This library also offers paging functionality. Collections such as calls and messages have `ListXxx` and `StreamXxx`
+functions that page under the hood. With both list and stream, you can specify the number of records you want to receive (limit) and the maximum size you want each page fetch to be (pageSize). The library will then handle the task for you.
+
+`List` eagerly fetches all records and returns them as a list, whereas `Stream` streams the records and lazily retrieves
+the pages as you iterate over the collection. You can also page manually using the `PageXxx` function in each of the apis.
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/twilio/twilio-go"
+	openapi "github.com/twilio/twilio-go/rest/api/v2010"
+	"os"
+)
+
+func main() {
+	accountSid := os.Getenv("TWILIO_ACCOUNT_SID")
+	authToken := os.Getenv("TWILIO_AUTH_TOKEN")
+	from := os.Getenv("TWILIO_FROM_PHONE_NUMBER")
+	
+	client := twilio.NewRestClient(accountSid, authToken)
+
+	params := &openapi.ListMessageParams{}
+	params.SetFrom(from)
+	params.SetPageSize(20)
+	limit := 100
+	
+	resp, err := client.ApiV2010.ListMessage(params, &limit)
```

Comment:
Prefer to have the `limit` are not be a pointer and assume 0-val means no limit. Then you don't need to store the var before calling this.

Reply:
Wanted to do that initially but then thought it would be weird to have the user pass in a `0` to mean no limit but I guess that's more user friendly.